### DISCUSSION TITLE
Access site on external devices for debugging

### DISCUSSION THREAD

**Discussion by tpkahlon** [vercel/turborepo#829] [Help] [answered]
As with [NextJS](https://nextjs.org/docs/api-reference/cli#development), we can use command like `next dev -p 3001 -H 192.168.0.14` to access a site on external devices on the same network to perform quick UI testing where `192.168.0.14` is our local IP.

Testing above steps on any site in Mono-repo results in a blank page on local port.

**Comment by anthonyshew** [comment, marked as answer]
If you're still seeing this problem, please open an issue!
